### ISSUE TITLE
RFC: Create a test module only used for testing the Forge publishing workflow

### ISSUE THREAD

**Issue by juniorsysadmin** [voxpupuli/plumbing#48]
Currently we don't know if our release process is broken until it comes time to release.
I suggest we have a module voxpupuli/release_testing which simply does an actual publish to the Forge as part of Travis testing

**Comment by bastelfreak** [comment]
good idea. I introduced a travis test to our matrix which tests if a module can build. BUt having an actual dummy module for such tests including the deployment is even better.

**Comment by nibalizer** [comment]
+1

**Comment by danzilio** [comment]
wish we could mark modules as hidden on the forge for stuff like this...

**Comment by binford2k** [comment]
if you delete all releases of a module, it's effectively hidden.

**Comment by bastelfreak** [comment]
We have https://forge.puppet.com/modules/puppet/example/readme now, and https://github.com/voxpupuli/gem-workflow-test.
